Cloud Release Notes since October 2024 (#7033)
umbraco/UmbracoDocs: umbraco-cloud/release-notes/overview-2024/2024-11-releasenotes.md

# November 2024

## Key Takeaways

> * **Umbraco 15** - Umbraco 15 and .NET 9 now available on Umbraco Cloud

Reviewer: ```suggestion
* **Umbraco 15** - Umbraco 15 and .NET 9 are now available on Umbraco Cloud.
```

> * **Web Application Firewall now available** - enabled by default on new environments. Existing projects can enable it via the portal.

Reviewer: ```suggestion
* **Web Application Firewall now available** - This feature is enabled by default in new environments. Existing projects can enable it through the portal.
```

## Umbraco 15
You can now start new projects on Umbraco 15 or upgrade your existing projects to take advantage of the latest features and improvements.

> Along with this, support for .NET 9 is now available on Umbraco Cloud, making it easy to stay up to date with the latest Microsoft tech stack.

Reviewer: ```suggestion
Support for .NET 9 is now available on Umbraco Cloud, simplifying the process of staying current with the latest Microsoft tech stack.
```

## Web Application Firewall now available
Web Application Firewall (WAF) is now enabled by default for all new environments on Umbraco Cloud.

> It helps protect your project against common threats like SQL injections, JavaScript exploits, and known vulnerabilities,
> using Cloudflare’s managed ruleset.

Reviewer: ```suggestion
It protects your project from common threats like SQL injections, JavaScript exploits, and known vulnerabilities, utilizing Cloudflare’s managed ruleset.
```

> For existing projects, WAF can be enabled via the Umbraco Cloud Portal. We recommend enabling it on your existing projects for added protection.

Reviewer: ```suggestion
For existing projects, WAF can be enabled through the Umbraco Cloud Portal. It is recommended to enable it on existing projects for added protection.
```
